tutor4points/core/templates/home.html

```diff
@@ -44,14 +44,21 @@ <h2>Tutor Requests</h2>
                                         {{tutor_request.tutor_date}} @<br>{{tutor_request.tutor_time}} <br/>
                                     </div>
                                     <div class = "respond-btns d-flex">
-                                        <button type="submit" name="accept-request" class="btn-secondary respond-button" 
-                                            data-toggle= "modal" data-target="#modal-accept-request">
-                                            Accept
-                                        </button>
-                                        <button type="submit" name="decline-request" class="btn-secondary respond-button" 
-                                        data-toggle= "modal" data-target="#modal-decline-request">
-                                            Decline
-                                        </button>
+                                        {% if tutor_request.accepted %}
+                                            <button type="submit" name="btn-accept-request" cless="btn-secondary"
+                                                data-toggle="modal" data-target="#modal-complete-request">
+                                                Complete 
```

Comment:
I think a better button name would be "Completed and Paid" or something along that line, so the tutor knows that they should only click the button after they have fully complete and paid for their session. Another option is to just add a checkbox to the form as "Paid" and "Completed" and make sure the user check marks those 2 fields before submitting the form.